src/BenchmarkDotNet/Loggers/LoggerExtensions.cs

```diff
@@ -20,7 +21,9 @@ public static class LoggerExtensions
 
         public static void WriteLineHint(this ILogger logger, string text) => logger.WriteLine(LogKind.Hint, text);
 
-        public static void Write(this ILogger logger, string text) =>  logger.Write(LogKind.Default, text);
+        public static void Write(this ILogger logger, string text) => logger.Write(LogKind.Default, text);
+
+        public static void WriteLineValidationError(this ILogger logger, ValidationError validationError) => logger.WriteLineError(validationError.Message);
```

Comment:
It's redundant since it will only be used in one place (when you remove `Logger` from `Validate`), it's best not to duplicate methods.

Reply:
@emanuel-v-r When the `Toolchain.InvalidCliPath(...) == false` you don't return actual error, so it's in the scope 😊

The new version of BDN should be out soon (1-2 weeks).
But it looks unfinished for the release (IMO)

```С#
public interface IToolchain
{
    bool IsSupported(BenchmarkCase benchmarkCase, ILogger logger, IResolver resolver);
    IEnumerable<ValidationError> Validate(BenchmarkCase benchmarkCase, ILogger logger, IResolver resolver);
}
```

```C#
var validationError = new ValidationError(true, "...", benchmarkCase);
logger.WriteLineValidationError(validationError);
return validationError.AsEnumerable();
```

And it would be cool, if you add a few tests that hits, for example, `Toolchain.InvalidCliPath` path.

Reply:
I am talking about removing logger dependencies from the `Toolchain` implementations.
As you mentioned `Toolchain.InvalidCliPath` is in scope, and it was addressed in scope of your other comment https://github.com/dotnet/BenchmarkDotNet/pull/2148#discussion_r995925947.
Here's the commit https://github.com/dotnet/BenchmarkDotNet/pull/2148/commits/40ae83c481bca11bc5855852ee7e203eb87f7c7a